app/Http/Controllers/CreatePostingPeriod.php

```
@@ -0,0 +1,184 @@
+                'success' => true,
+                'message' => 'Posting Period Successfully Created',
+                'data' => [
+                    'posting_period' => $postingPeriod->fresh(['periods']),
+                    'period' => $period,
+                    'create_for_month' => $nextMonth->format('F Y'),
+                    'execution_date' => $currentDate->toDateString(),
+                ],
+            ], 201); 
+            
+        } catch (\Exception $e) {
+            DB::rollBack();
```

PHPMD (2.15.0)

Avoid using static access to class '\DB' in method '__invoke'. (Clean Code Rules)

(StaticAccess)